cmd/sebak/cmd/run.go

```diff
@@ -45,18 +46,28 @@ var (
 	flagTLSKeyFile          string = sebakcommon.GetENVValue("SEBAK_TLS_KEY", "sebak.key")
 	flagValidators          string = sebakcommon.GetENVValue("SEBAK_VALIDATORS", "")
 	flagThreshold           string = sebakcommon.GetENVValue("SEBAK_THRESHOLD", "66")
+	flagTimeoutINIT         string = sebakcommon.GetENVValue("TIMEOUT_INIT", "2")
+	flagTimeoutSIGN         string = sebakcommon.GetENVValue("TIMEOUT_SIGN", "2")
+	flagTimeoutACCEPT       string = sebakcommon.GetENVValue("TIMEOUT_ACCEPT", "2")
+	flagTimeoutALLCONFIRM   string = sebakcommon.GetENVValue("TIMEOUT_ALLCONFIRM", "2")
+	flagTransactionsLimit   string = sebakcommon.GetENVValue("TRANSACTIONS_LIMIT", "1000")
```

Comment:
Sorry I didn't see it earlier, but all config have a SEBAK prefix, which I think is worth it, as those names are pretty generic.